src/DynamoCoreWpf/ViewModels/Watch3D/shaderSource/vsDynamoPointLine.hlsl

```diff
@@ -5,17 +5,17 @@
 #include"DataStructs.hlsl"
 #pragma pack_matrix( row_major )
 
-PSInputPS main(VSInputPS input)
+GSInputPS main(VSInputPS input)
 {
-    PSInputPS output;
+	GSInputPS output;
```

Comment:
Are you using the default geometry shader?

Reply:
Did you have to make any changes to `DynamoEffectsManager`?

Reply:
I already added the geometry shader in the last commits that you asked me to commit directly to your branch.

Reply:
also, I used the point geometry shader that helix uses by default - for lines we'll need to add another shader and use the line geometry shader from helix.